homeassistant/components/pi_hole/__init__.py

```diff
@@ -97,13 +99,19 @@ def update_unique_id(
     await er.async_migrate_entries(hass, entry.entry_id, update_unique_id)
 
     session = async_get_clientsession(hass, verify_tls)
-    api = Hole(
-        host,
-        session,
-        location=location,
-        tls=use_tls,
-        api_token=api_key,
-    )
+    hole_kwargs = {
+        "host": host,
```

Comment:
host and location are positional arguments but I call them here in kwargs for simplicity